internal/cli/config.go

```diff
@@ -82,6 +86,14 @@ func (opts *configOpts) Run() error {
 				Help:    helpLink,
 			},
 		},
+		{
+			Name: "projectId",
+			Prompt: &survey.Input{
+				Message: "Project ID [You can leave it blank but remember to use the flag --projectId]:",
+				Help:    helpLink,
```

Comment:
`helpLink` is a link to the API docs, it won't be useful here so probably the long description you have above could be better here, something like, `This is the ID or an existing project your API keys have access to,  You can leave this blank and specify it on every command with  --projectId`